packages/theme/components/ValidationProviders/EmailValidationProvider.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+  <ValidationProvider
+    v-slot="{ errors }"
+    :rules="mergedRules"
+  >
+    <slot
+      :errors="errors"
+    />
+  </ValidationProvider>
+</template>
+
+<script>
+import { ValidationProvider } from 'vee-validate';
+import { computed } from '@nuxtjs/composition-api';
+
+export default {
+  name: 'EmailValidationProvider',
+  props: {
+    name: {
+      type: String,
+      default() {
+        return 'email';
+      }
+    },
+    rules: {
+      type: String,
+      default() {
+        return 'email';
+      }
+    },
+    'append-rules': {
+      type: String,
+      default() {
+        return '';
+      }
+    }
+  },
+  components: {
+    ValidationProvider
+  },
+  setup({ rules, appendRules }) {
```

Comment:
Props shouldn't be destructured to avoid losing reactivity.

https://vuejs.org/api/composition-api-setup.html#accessing-props
https://eslint.vuejs.org/rules/no-setup-props-destructure.html

Reply:
Got it!